README.Rmd

```diff
@@ -49,18 +50,24 @@ devtools::install_github("UBC-MDS/rpollution")
 
 ## Usage
 
-- TODO
-
-## Contributing
+1. Create an [OpenWeather API Key](https://openweathermap.org/appid)
+2. Install `rpollution`
 
-Interested in contributing? Check out the contributing guidelines. Please note that the rpollution project is released with a [Contributor Code of Conduct](https://github.com/UBC-MDS/rpollution/blob/master/CODE_OF_CONDUCT.md). By contributing to this project, you agree to abide by its terms.
-
-Contributors
+To use the package, import the package with following commands:
```

Comment:
To use the package, import the package with the following command:

Reply:
cool, fixed